common-gradle-enterprise-maven-configuration/.mvn/gradle-enterprise.xml

```diff
@@ -32,16 +32,19 @@
       <enabled>true</enabled>
     </local>
     <remote>
+     <!-- By default, using the builtin Gradle Enterprise build cache node and access key -->
```

Comment:
i think for maven since maven requires Gradle Enterprise authentication, we can remove the `server` and `credentials` sections here. WDYT @etiennestuder ?